### PR TITLE
created vishalchauhan.json

### PR DESCRIPTION
**Pull request by vish-han**
## Changes proposed

I added vishalchauhan.json

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1188"></a>

### CODE REVIEW

File: public/data/vishalchauhan.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Vishal Chauhan",
+  "type": "personal",
+  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies",
+  "avatar": "https://avatars.githubusercontent.com/u/44545745?v=4",
```
Comment (vikasganiga05) [inline public/data/vishalchauhan.json:5]:
```suggestion
  "avatar": "https://github.com/vish-han.png",
```
Please follow [To Add Your Profile](https://github.com/EddieHubCommunity/LinkFree#-to-add-your-profile) documentation. Replace the avatar with this URL format :point_right: `https://github.com/<yourusername>.png`.

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Please change the avatar URL to :point_right: [https://github.com/vish-han.png](https://github.com/vish-han.png)

**Review by krshdev — APPROVED** [review]
Looks good, except the gh profile... Just that and done

**Review by kumarsonsoff3 — APPROVED** [review]
𝗟𝗼𝗼𝗸𝘀 𝗚𝗼𝗼𝗱 𝘁𝗼 𝗺𝗲! 👍  𝗧𝗵𝗮𝗻𝗸 𝘆𝗼𝘂 𝗳𝗼𝗿 𝘆𝗼𝘂𝗿 𝗰𝗼𝗻𝘁𝗿𝗶𝗯𝘂𝘁𝗶𝗼𝗻 🚀 😊

**Review by schmelto — APPROVED** [review]
Lgtm

**Review by krshdev — APPROVED** [review]

**Review by ssensalo — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!,

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**